frontend/src/metabase/lib/moderation.js

```diff
@@ -0,0 +1,10 @@
+export function getModerationStatusIcon(status) {
+  switch (status) {
+    case "verification":
+      return "verified";
+    case "flag":
+      return "warning_colorized";
+    case "question":
+      return "clarification";
+  }
+}
```

Comment:
nip: I usually find objects pretty handy in cases like that and they add more declarative flavor to the code. 

Example:

```js
const statusIconMap = {
   verification: "verified",
   flag: "warning_colorized",
   question: "clarification",
}
return statusIconMap[status];
```

Reply:
nice, I'll take it a step further to skip having to name a variable